source/neuropod/internal/neuropod_tensor.hh

```diff
@@ -460,7 +514,9 @@ public:
     static int cmp(const std::string &lhs, const std::string &rhs) { return lhs.compare(rhs); }
 
     // Based on https://en.cppreference.com/w/cpp/language/operators
+    // TODO(vip): Simplify these operators
```

Comment:
Maybe in C++ 20 https://devblogs.microsoft.com/cppblog/simplify-your-code-with-rocket-science-c20s-spaceship-operator/